data/incipias/incipias ships.txt

```diff
@@ -0,0 +1,239 @@
+
+ship "Nimbus"
+	"plural" "Nimbi"
+	"sprite" ship/nimbus
+	"thumbnail" thumbnail/nimbus
+	"swizzle" 0
+	attributes
+		"category" "Medium Warship"
+		licenses
+			"Hicemus Conflict"
+		"cost" 500000
+		"hull" 5000
+		"shields" 7500
+		"required crew" 20
+		"bunks" 40
+		"mass" 1640
+		"drag" 10.8
+		"heat dissipation" .46
+		"cargo space" 75
+		"outfit space" 450
+		"weapon capacity" 120
+		"engine capacity" 85
+		"gaslining" 1
+		"fuel capacity" 200
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 4
+		"Stagnation Beam"
+		"Cloud Piercer" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+	engine 0 120
+	engine 0 120
+	engine -30.5 120
+	engine 30.5 120
+	gun -15 -117
+	gun 15 -117
+	gun 0 -117
+	gun 0 -117
+	turret 0 31.5
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The Nimbus is the first spaceship that the Incipias built that was truly able to navigate through a system. Since it was first manufactured it has been cherished among wealthier captains and a dream of captains just starting out."
+	description "Although the Nimbus is labeled as a medium warship, it is multipurpose; whether one uses it for mining, passenger transport or cargo, it will perform its job nicely. It is only occasionally used in combat because the Incipias are not currently fighting open wars."
+
+ship "Nimbus" "Nimbus (Civil)"
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+ship "Nimbus" "Nimbus (Mining)"
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 4
+		"Stagnation Beam" # to be replaced with a flotsam tractor beam
+		"Cloud Piercer" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+ship "Nimbo Stratus"
+	"plural" "Strati"
+	"sprite" ship/stratus
+	"thumbnail" thumbnail/stratus
+	"swizzle" 0
+	attributes
+		"category" "Utility"
+		"cost" 50000
+		"hull" 500
+		"shields" 700
+		"required crew" 5
+		"bunks" 10
+		"mass" 534
+		"drag" 8.4
+		"heat dissipation" .67
+		"cargo space" 50
+		"outfit space" 270
+		"weapon capacity" 50
+		"engine capacity" 60
+		"gaslining" 1
+		"fuel capacity" 200
+
+	outfits
+		"Gas-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 2
+
+		"MH Blue-Class Engine"
+		"MH Blue-Class Steering"
+
+	gun -13 -48.5
+	gun 13 -48.5
+	engine 0 61.5
+	explode "medium explosion" 10
+	explode "large explosion" 20
+	"final explode" "final explosion medium"
+	description "Mining has always been a big business in Incipias society, which is why the Stratus is the most common ship for beginning captains. Its cheap price tag and ability to bring money in quickly ensure its continuing popularity. While the Stratus could theoretically be used in combat, it stands no chance against the Nimbus and therefore has never been used by the Hicemus military forces."
```

Comment:
```suggestion
	description "Mining has always been a big business in the Incipia society, which is why the Stratus is the most common ship for beginning captains. Its cheap price tag and ability to bring money in quickly ensure its continuing popularity. While the Stratus could theoretically be used in combat, it stands no chance against the Nimbus and therefore has never been used by the Hicemus military forces."
```

Reply:
no they are called Incipias

Reply:
I assumed that was a plural, so I have been looking at it wrong. Probably resolve this.